addons/test_website_modules/static/tests/tours/configurator_flow.js

```diff
@@ -18,12 +18,13 @@ tour.register('configurator_flow', {
         run: 'text Website Test',
     }, {
         content: "validate the website creation modal",
-        trigger: 'button.btn-primary',
+        trigger: 'button.btn-primary[name="create_and_redirect_configurator"]',
```

Comment:
Is this change necessary (you don't talk about it in your commit)? Are they multiple buttons matching this selector?